rw-system.sh

```diff
@@ -274,6 +274,14 @@ changeKeylayout() {
         changed=true
     fi
 
+    if getprop ro.vendor.build.fingerprint | grep -iq -e motorola/liber; then
+      if [ ! -f /mnt/phh/keylayout/uinput_nav.kl ]; then
```

Comment:
Why this if?

Reply:
I thought the file was going to be kept on disk, I'll remove it